bin/main.js

```diff
@@ -12,6 +12,7 @@ var AWS         = require('aws-sdk-promise');
 // These will exist in taskcluster-base, when we move to the next version.
 var config      = require('typed-env-config');
 var loader      = require('taskcluster-lib-loader');
+var tc_lib_app  = require('taskcluster-lib-app');
```

Comment:
I just shipped a new version of it, and I think we are getting away from bundling everything in base, right?